pre_commit_hooks/loaderon_hooks/odoo_specific_hooks/check_model_name.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
+import os
+import re
+
+from pre_commit_hooks.loaderon_hooks.util.template_methods.file_bunches_lines_checker_template_method import \
+    FileBunchesLinesCheckerTemplateMethod
+
+
+class ModelNameAttributeChecker(FileBunchesLinesCheckerTemplateMethod):
+    def __init__(self, argv):
+        super(ModelNameAttributeChecker, self).__init__(argv)
+        self.__module_name = ''
+        name_attribute_regex = r'^(\t| )*_name = .+'
+        self.__name_pattern = re.compile(name_attribute_regex)
+        inherit_attribute_regex = r'^(\t| )*_inherit = .+'
+        self.__inherit_pattern = re.compile(inherit_attribute_regex)
+        self.__name_line = ''
+        self.__inherit_line = ''
```

Comment:
I haven't seen it written anywhere but I tend to only use double-underscore when I expect someone to inherit from my class and I need to avoid clashing their namespace.  It doesn't really provide any further "protection" from accesses, just a name mangling with the class (`__foo` becomes `_C__foo`)